src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
Better use `enum` here - it will give you more control and flexibility for `Shape` type.